com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs

```diff
@@ -116,10 +116,11 @@ public static void SetViewAndProjectionMatrices(CommandBuffer cmd, Matrix4x4 vie
 
             if (setInverseMatrices)
             {
-                Matrix4x4 inverseMatrix = Matrix4x4.Inverse(viewMatrix);
-                // Note: inverse projection is currently undefined
+                Matrix4x4 inverseViewMatrix = Matrix4x4.Inverse(viewMatrix);
+                Matrix4x4 inverseProjectionMatrix = Matrix4x4.Inverse(projectionMatrix);
                 Matrix4x4 inverseViewProjection = Matrix4x4.Inverse(viewAndProjectionMatrix);
```

Comment:
Nitpick: Since we have inverse view and inverse proj now we can compute inverse(ViewProj) more efficiently than inverting viewAndProj? I suppose here inverse is expensive operation.

Reply:
Calculating inverse from matrices this way is definitely inaccurate.
I implemented motion vector support and temporal aliasing technique in the past, we found out that using using inverse matrices calculated this way was not accurate enough and did not allowed us to accurately re-project a pixel coordinate in the previous frame. Instead, when we calculated the view and projection matrices, we would also calculate their inverses using the same input parameters. For instance, we would have equivalent Unity functions:
```
Matrix4x4.PerspectiveInverse(float fov, float aspect, float zNear, float zFar)
Matrix4x4.LookAtInverse(Vector3 from, Vector3 to, Vector3 up)
```

Reply:
I removed the extra inverse.